PWGCF/Flow/Tasks/flowGFWOmegaXi.cxx

```diff
@@ -557,24 +680,44 @@ struct FlowGFWOmegaXi {
       auto v0negdau = v0.negTrack_as<DaughterTracks>();
       // check tpc
       int PDGCode = 0;
-      if (v0.qtarm() / TMath::Abs(v0.alpha()) > cfgv0_ArmPodocut && TMath::Abs(v0posdau.tpcNSigmaPi()) < cfgNSigmaCascPion && TMath::Abs(v0negdau.tpcNSigmaPi()) < cfgNSigmaCascPion && TMath::Abs(v0posdau.tofNSigmaPi()) < cfgNSigmaCascPion && TMath::Abs(v0negdau.tofNSigmaPi()) < cfgNSigmaCascPion) {
+      // fill QA
+      registry.fill(HIST("hqaarm_podobefore"), v0.alpha(), v0.qtarm());
+      // check daughter TPC and TOF
+      if (v0.qtarm() / TMath::Abs(v0.alpha()) > cfgv0_ArmPodocut &&
+          (!cfgcheckDauTPC || (TMath::Abs(v0posdau.tpcNSigmaPi()) < cfgtpcNSigmaCascPion && TMath::Abs(v0negdau.tpcNSigmaPi()) < cfgtpcNSigmaCascPion)) &&
+          (!cfgcheckDauTOF || (TMath::Abs(v0posdau.tofNSigmaPi()) < cfgtofNSigmaCascPion && TMath::Abs(v0negdau.tofNSigmaPi()) < cfgtofNSigmaCascPion))) {
```

Comment:
It would be good to start moving to `std::fabs`
O2Physics workflows are `c++` not `ROOT` programs

Reply:
Thanks for your comment.